src/System.Device.Gpio/System/Device/Gpio/RaspberryBoardInfo.cs

```diff
@@ -199,7 +196,7 @@ private Model GetBoardModel()
         /// <value>
         /// The name of the processor.
         /// </value>
-        public string ProcessorName
+        public string? ProcessorName
```

Comment:
I'd vote to refactor a code a bit to make this non-null. Treat as optional considering it's internal code

Reply:
I wasn't sure how to approach that one. I switched to assigning `string.Empty` as a fallback.